packages/babel-preset-react/README.md

```diff
@@ -5,7 +5,6 @@
 This preset always includes the following plugins:
 
 - [syntax-jsx](https://babeljs.io/docs/plugins/syntax-jsx/)
-- [transform-flow-strip-types](https://babeljs.io/docs/plugins/transform-flow-strip-types/)
```

Comment:
Wonder if it's worth including a note about how Flow syntax support is no longer enabled in 7.x?